execute/table/chunk.go

```diff
@@ -126,6 +126,12 @@ func (v Chunk) Strings(j int) *array.String {
 	return v.Values(j).(*array.String)
 }
 
+// Times is a convenience function for retrieving an array
+// of timestamps as integers
+func (v Chunk) Times(j int) *array.Int {
```

Comment:
Is this needed? It's identical to `Ints` and there's no check for the column type.